grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -231,6 +234,14 @@ class Location implements Comparable<Location>, java.io.Serializable {
         return supportsAny(requiredActivities) && !binLocations?.empty
     }
 
+    Boolean isInternalLocation() {
+        return locationType?.locationTypeCode in LocationTypeCode.listInternalTypeCodes()
```

Comment:
So if you add isInternalLocation to LocationType, then this method would probably look something like 
```
return locationType?.isInternalLocation()
```
and LocationType.isInternalLocation() would look like this
```
return locationTypeCode in LocationTypeCode.listInternalTypeCodes()
```

Reply:
Correct?